packages/gravity-ui-nunjucks/gulp/fractal-tasks.js

```diff
@@ -0,0 +1,61 @@
+const fractal = require('../fractal');
+
+const taskNamePrefix = 'fractal:';
+
+// keep a reference to the fractal CLI console utility
+const logger = fractal.cli.console;
+
+/*
+ * Start the Fractal server
+ *
+ * In this example we are passing the option 'sync: true' which means that it will
+ * use BrowserSync to watch for changes to the filesystem and refresh the browser automatically.
+ * Obviously this is completely optional!
+ *
+ * This task will also log any errors to the console.
+ */
+async function startServer() {
+  const server = fractal.web.server({
+    sync: true,
+  });
+
+  server.on('error', (err) => {
+    return logger.error(err.message);
+  });
+
+  await server.start();
+  logger.success(`Fractal server is now running at ${server.url}`);
+  logger.success(`Network URL: ${server.urls.sync.external}`);
+}
+startServer.displayName = `${taskNamePrefix}start`;
+
+
+/*
+ * Run a static export of the project web UI.
+ *
+ * This task will report on progress using the 'progress' event emitted by the
+ * builder instance, and log any errors to the terminal.
+ *
+ * The build destination will be the directory specified in the 'builder.dest'
+ * configuration option set above.
+ */
+async function buildPatternLibrary() {
+  const builder = fractal.web.builder();
+
+  builder.on('progress', (completed, total) => {
+    return logger.update(`Exported ${completed} of ${total} items`, 'info');
+  });
+  builder.on('error', (err) => {
+    return logger.error(err.message);
+  });
+
+  await builder.build();
+  logger.success('Fractal build completed!');
+}
+buildPatternLibrary.displayName = `${taskNamePrefix}build`;
+
+
```

Comment:
Please include a lint check for concurrent whitespace.

Reply:
I've just rebased on the current `next` branch, which contains your recent, shared ESLint setup. I've fixed all the stuff that was being flagged - including the double blank lines - and it's oh so pretty now! ^_^